examples/jenkins/pipeline/pipelinetemplate.json

```diff
@@ -19,6 +19,9 @@
         "creationTimestamp": null,
         "labels": {
           "name": "sample-pipeline"
+        },
+        "annotations": {
+          "downstream.openshift.io/deployments": "[{\"name\": \"frontend\", \"namespace\": \"\"}]"
```

Comment:
`deployments.openshift.io/will-trigger` maybe.  This is an annotation that indicates that a deployment will be triggered by this PR.

I don't love that it's deployment specific.  What's the equivalent annotation for builds?  Do we want the same one?  

This maybe seems like a reverse service dependency.